lib/contentful/sys_data.ex

```diff
@@ -1,15 +1,23 @@
 defmodule Contentful.SysData do
+  alias Contentful.ContentType
+
   @moduledoc """
   The SysData represents internal additional data for Contetnful API objects, usually found in the
   "sys" part of the response objects. It's also referred to as "common properties".
 
   See the [official documentation for more information](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/locales).
   """
-  defstruct [:id, :revision, :version]
+  defstruct [:id, :revision, :version, :created_at, :updated_at, locale: nil, content_type: nil]
 
   @type t :: %Contentful.SysData{
           id: String.t(),
           revision: integer() | nil,
-          version: integer() | nil
+          version: integer() | nil,
+          created_at: String.t() | nil,
+          updated_at: String.t() | nil,
```

Comment:
Not sure about this one, might be optionally present I guess.

Reply:
According to https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/common-resource-attributes, this should be present for `Entry`, `Asset` and `ContentTypes`.

Reply:
That also explicitly excludes `Space`, it seems.